<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchReader</class>
 <widget class="QWidget" name="MatchReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Memory Reader</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="done">
          <property name="text">
           <string>SEN:P-AI noticed you!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>|</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="doing">
          <property name="text">
           <string>[In Progress]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="bruteForce">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Try Harder</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="_0">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>133</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="showTime">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>show time</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="qrCode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>29</width>
          <height>29</height>
         </size>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Jan 0, 00:00:00.000 UTC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>qrCode</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
